src/packagedcode/cocoapods.py

```diff
@@ -0,0 +1,193 @@
+# All rights reserved.
+# http://nexb.com and https://github.com/nexB/scancode-toolkit/
+# The ScanCode software is licensed under the Apache License version 2.0.
+# Data generated with ScanCode require an acknowledgment.
+# ScanCode is a trademark of nexB Inc.
+#
+# You may not use this software except in compliance with the License.
+# You may obtain a copy of the License at: http://apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+#
+# When you publish or redistribute any data created with ScanCode or any ScanCode
+# derivative work, you must accompany this data with the following acknowledgment:
+#
+#  Generated with ScanCode and provided on an "AS IS" BASIS, WITHOUT WARRANTIES
+#  OR CONDITIONS OF ANY KIND, either express or implied. No content created from
+#  ScanCode should be considered or used as legal advice. Consult an Attorney
+#  for any legal advice.
+#  ScanCode is a free software code scanning tool from nexB Inc. and others.
+#  Visit https://github.com/nexB/scancode-toolkit/ for support and download.
+
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import logging
+import re
+
+import attr
+from packageurl import PackageURL
+
+from commoncode.fileutils import py2
+from commoncode import filetype
+from commoncode import fileutils
+from packagedcode import models
+from packagedcode import spec
+
+
+"""
+Handle cocoapods packages manifests for macOS and iOS
+including .podspec, Podfile and Podfile.lock files.
+See https://cocoapods.org
+"""
+
+# TODO: implementation to get dependency data using gemsfileparser
```

Comment:
If you keep these for another PR later, it could make sense to enter a ticket?

Reply:
No, I'm not keeping these for another PR. Actually at that time we were using gemfileparser library to get the data and in that library dependencies are fetched which are in a particular format only. 
I will use the gemfileparser2 library here and solve this :)